src/components/header/template.njk

```diff
@@ -1,105 +1,135 @@
-          </svg>
-          {% if (params.productName) %}
-          <span class="govuk-header__product-name">
-            {{ params.productName }}
-          </span>
-          {% endif %}
-        </a>
-      </div>
-      {% if params.serviceName or params.navigation | length or params.signOutHref or params.languageToggle %}
-      <div class="govuk-header__content">
-      {% if params.serviceName %}
-        {% set serviceNameTag = 'a' if params.serviceUrl else 'span' %}
-        <{{ serviceNameTag }} {% if (serviceNameTag == 'a') %}href="{{ params.serviceUrl }}" {% endif %} class="hmrc-header__service-name {% if (serviceNameTag == 'a') %}hmrc-header__service-name--linked{% endif %}">
-          {{ params.serviceName }}
-        </{{ serviceNameTag }}>
+{%- set menuButtonText = 'Dewislen' if language == 'cy' else 'Menu' -%}
+
+{%- set _stEdwardsCrown %}
```

Comment:
apologies Tim, I missed this - the underscore is because I've copied the implementation from the govuk header and that's how they've done it- I'm not sure what the significance is for them (maybe that it's temporary / will go away soon - but not sure why that would be important to convey here because I don't think the variables would be accessible anywhere else)

Reply:
No problem, was just curious. I thought there'd be a reason even if only to keep consistency from govuk-frontend